.projenrc.ts

```diff
@@ -74,4 +74,14 @@ new FunctionlessProject({
   cdkVersion: "2.39.1",
 });
 
+new FunctionlessProject({
+  parent: turbo,
+  defaultReleaseBranch: "main",
+  name: "cron",
+  outdir: "packages/cron",
+  cdkVersion: "2.39.1",
+  devDeps: ["@types/node-fetch"],
+  deps: ["node-fetch"],
+});
```

Comment:
Curious if we should use the template instead?

Reply:
Push this with projen and then eject the whole package later?

Reply:
Is it hard to just run yarn create functionless and move the readme and code into it? I also noticed that the readme talked about the template. If it's a pain that's fine but if not I would vote do it now.

Reply:
I think the current turbo harness relies on projects running projen. I think all or nothing here is the best. Lets eject everything at the same time.